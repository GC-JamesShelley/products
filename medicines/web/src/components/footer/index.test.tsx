import { shallow } from 'enzyme';
import React from 'react';
import Footer from './index';

describe(Footer, () => {
  it('should render', () => {
    const component = shallow(<Footer />);
    expect(component).toMatchSnapshot();
  });
});
