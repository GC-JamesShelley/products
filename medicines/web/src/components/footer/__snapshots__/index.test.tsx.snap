// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render 1`] = `
<styled.footer>
  <div
    className="wrapper"
  >
    <picture>
      <SvgAgencyDigitalLogo />
    </picture>
    <nav>
      <ul>
        <li>
          <p>
            <Link
              href="/cookies"
            >
              <a>
                Cookie Policy
              </a>
            </Link>
          </p>
        </li>
        <li>
          <p>
            <Link
              href="https://www.gov.uk/government/publications/mhra-privacy-notice/mhra-privacy-notice"
            >
              <a>
                Privacy Policy
              </a>
            </Link>
          </p>
        </li>
        <li>
          <p>
            <Link
              href="/accessibility"
            >
              <a>
                Accessibility Statement
              </a>
            </Link>
          </p>
        </li>
        <li>
          <p>
            <Link
              href="/about"
            >
              <a>
                About this service
              </a>
            </Link>
          </p>
        </li>
      </ul>
    </nav>
  </div>
</styled.footer>
`;
